Trappist/src/Promact.Trappist.Web/wwwroot/app/tests/tests.service.ts

```diff
@@ -29,4 +29,21 @@ export class TestService {
     IsTestNameUnique(testName: string, id: number) {
         return this.httpService.get(this.testNameApiUrl + '/' + testName + '/' + id);
     }
+
+    /**
+     * Gets the Settings saved for a particular Test
+     * @param id is used to get the Settings of a Test by its Id
+     */
+    getSettings(id: number) {
```

Comment:
Settings are specific to Test. Please change name accordingly at all require places.